tools/deprecate_guards/deprecate_guards.py

```diff
@@ -180,7 +180,7 @@ async def create_issue(self, title: str, body: str, login: str) -> None:
                 else:
                     self.log.warning(f"Dry run: not creating issue '{title}' for {login}")
                 return
-        except gidgethub.Exception:
+        except gidgethub.GithubException:
```

Comment:
```suggestion
        except gidgethub.GitHubException:
```

argh - if we shift this